sw/device/tests/spi_device_ottf_console_test.c

```diff
@@ -0,0 +1,114 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_spi_device.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+OTTF_DEFINE_TEST_CONFIG(.console.type = kOttfConsoleSpiDevice,
+                        .console.base_addr = TOP_EARLGREY_SPI_DEVICE_BASE_ADDR,
+                        .console.test_may_clobber = false, );
+
+static const char test_str[] = "This is a SPI device OTTF console test.";
```

Comment:
```suggestion
static const char kTestStr[] = "This is a SPI device OTTF console test.";
```
see https://google.github.io/styleguide/cppguide.html#Constant_Names

applies for others below too

Reply:
Renamed these variables. Thanks for the reference.